src/constants/app-paths.ts

```diff
@@ -0,0 +1,11 @@
+enum AppPaths {
```

Comment:
check pls differences between `enum` and `const enum`

Reply:
checked